installers/Launcher.hs

```diff
@@ -39,8 +39,12 @@ launcherArgs launcher = unwords $
         "--kademlia-peers-file", "ip-dht-mappings",
         "--system-start", "1499360281",
         "--wallet",
+        "--tlscert", quote (runtimePath launcher <> tlsPath <> "server" <> pathSeparator <> "server.crt"),
```

Comment:
@deepfire I was not sure is this correct path based on path where certificates are created. Can you check this?